services/path.py

```diff
@@ -165,6 +171,19 @@ def __init__(
                 if search_term.lower() in path.relative_path.lower()
             ]
 
+    @cached_property
+    def files_accounting_flags(self) -> List[str]:
```

Comment:
I think you could just call this `files` (which takes into account it's own state).  Outside callers don't necessarily care about the flag part (other than passing them into the constructor).